fastplotlib/layouts/_base.py

```diff
@@ -420,6 +424,12 @@ def auto_scale(self, maintain_aspect: bool = False, zoom: float = 0.8):
         else:
             width, height, depth = (1, 1, 1)
 
+        # make sure width and height are non-zero
```

Comment:
depth not being used here?

Reply:
we need to start doing 3D stuff to properly learn how the depth works.